src/utils/structures.js

```diff
@@ -11,11 +18,18 @@ const combine = (target, source, isDeep) =>
 
 const combineDeep = (memo, source) => combine(memo, source, true);
 
+/**
+ * Shallow merge an array of objects. Earlier values overwritten by later values
+ * @param sources Array of objects and functions to be merged using `combine`
+ * @see combine
+ */
 module.exports.combine = (...sources) => sources.reduce(combine, {});
 
+/**
+ * Deep merge an array of objects. Earlier values overwritten by later values.
+ * @param sources Array of objects and functions to be merged using `combine`
+ * @see combine
+ */
 module.exports.merge = (...sources) => sources.reduce(combineDeep, {});
 
-module.exports.setOfKeysAndValues = source =>
```

Comment:
"setOfKeysAndValues" isn't referenced anywhere else in the project and it seems to run fine without it.